config/billing_rates.yml

```diff
@@ -1,20 +1,26 @@
-- { id: 1bde2200-545a-8d78-960e-08a303111e3d, resource_type: IPAddress,           resource_family: IPv4,            location: hetzner-hel1,   unit_price: 0.000074405 }
-- { id: 38685ec4-38eb-4460-aed3-8a36bab14175, resource_type: IPAddress,           resource_family: IPv4,            location: github-runners, unit_price: 0.000074405 }
-- { id: 36b3883a-a72a-4c08-a4c6-482f4f91e148, resource_type: PostgresCores,       resource_family: standard,        location: hetzner-fsn1,   unit_price: 0.001252425 }
-- { id: a5ef9efa-95eb-400b-a66a-31dd9d7a654a, resource_type: PostgresCores,       resource_family: standard,        location: hetzner-hel1,   unit_price: 0.001198557 }
-- { id: 10ac0c66-ad6f-49f9-9c89-f676501eacea, resource_type: PostgresCores,       resource_family: standard,        location: github-runners, unit_price: 0.001198557 }
-- { id: 80aa487a-0aa7-472f-a52c-d82a96067cba, resource_type: PostgresStorage,     resource_family: standard,        location: hetzner-fsn1,   unit_price: 0.000002810 }
-- { id: c41422cd-9e28-46bd-acd4-02debfcd26a2, resource_type: PostgresStorage,     resource_family: standard,        location: hetzner-hel1,   unit_price: 0.000002810 }
-- { id: b8d48f79-8131-46aa-b898-dba1c21d0954, resource_type: PostgresStorage,     resource_family: standard,        location: github-runners, unit_price: 0.000002810 }
-- { id: d772d0aa-0b40-4b7a-aceb-72f6211f7cad, resource_type: GitHubRunnerMinutes, resource_family: standard-2,      location: global,         unit_price: 0.0008 }
-- { id: d885c82d-b139-40d5-890f-de3720896b3b, resource_type: GitHubRunnerMinutes, resource_family: standard-4,      location: global,         unit_price: 0.0016 }
-- { id: e50e6bff-6ee9-493b-bc09-75f122863d08, resource_type: GitHubRunnerMinutes, resource_family: standard-8,      location: global,         unit_price: 0.0032 }
-- { id: 89cda6c2-2650-4ac1-bc28-deeae5e3b756, resource_type: GitHubRunnerMinutes, resource_family: standard-16,     location: global,         unit_price: 0.0064 }
-- { id: ab92083e-2be7-4038-b946-b060af3994fb, resource_type: GitHubRunnerMinutes, resource_family: standard-2-arm,  location: global,         unit_price: 0.0008 }
-- { id: 078d2f47-5827-49fa-9d13-bee6d1d8cadd, resource_type: GitHubRunnerMinutes, resource_family: standard-4-arm,  location: global,         unit_price: 0.0016 }
-- { id: 0fc7f264-87f0-446d-bee7-c0454149b1bc, resource_type: GitHubRunnerMinutes, resource_family: standard-8-arm,  location: global,         unit_price: 0.0032 }
-- { id: 995ab760-5d97-4a7b-981e-c9947233c78e, resource_type: GitHubRunnerMinutes, resource_family: standard-16-arm, location: global,         unit_price: 0.0064 }
+- { id: 139d9a67-8182-8578-a303-235cabd5161c, resource_type: VmCores,                resource_family: standard,        location: hetzner-fsn1,   unit_price: 0.000622768 }
```

Comment:
Most lines are just whitespace changes as I added spaces for aligning.

Reply:
This is a better view of the diff; https://www.diffchecker.com/e9TfQDnP/

Reply:
GitHub is also able to hide whitespaces
<img width="1498" alt="Screenshot 2024-02-13 at 16 31 06" src="https://github.com/ubicloud/ubicloud/assets/993199/a62a02cf-866b-402d-985c-009f6960624a">

Reply:
I was looking for this option thinking it is such a core feature, it must be somewhere, but couldn't find it :smile: